pootle/apps/accounts/views.py

```diff
@@ -7,16 +7,36 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
+import logging
+
 from django.core.urlresolvers import reverse
 from django.shortcuts import redirect
 
 from allauth.account.views import LoginView
+from allauth.exceptions import ImmediateHttpResponse
 from allauth.socialaccount.helpers import _add_social_account
 from allauth.socialaccount.models import SocialLogin
 
 from .forms import SocialVerificationForm
 
 
+logger = logging.getLogger(__name__)
+
+
+class PootleLoginView(LoginView):
+
+    def form_valid(self, form):
+        success_url = self.get_success_url()
+        try:
+            return form.login(self.request, redirect_url=success_url)
+        except ImmediateHttpResponse as e:
+            return e.response
+        except Exception as e:
+            logger.exception("%s %s" % (e.__class__.__name__, e))
```

Comment:
@unho after further investigation i was mistaken about how logging.exception worked.

I do however feel that this is a better error message for a a sysadmin because on the line which has ERROR it has what kind of error as well as the error message - although i accept this exception is there half way through the traceback.